cypress/support/elements/researcher_reports_page_elements.js

```diff
@@ -1,6 +1,77 @@
-const researcherReportsPageElements = {
-    BTN_GENERATE_CSV_REPORT_LINK: 'div.container div:nth-child(2) button',
-    LNK_TO_LAST_REPORT: 'div.container div:nth-child(2) div.item-info a',
-}
+const option = ["Schools", "Teachers", "Resources", "Permission forms"];
 
-export default  researcherReportsPageElements
\ No newline at end of file
+export const ResearcherReportsPageElements = {
+
+	getHeader() {
+		return cy.get('#form-container').prev();
+	},
+	verifyUserName() {
+		this.getHeader().should("contain", "joe user");
```

Comment:
The username, here "joe user", should be a 'username' input arg, not hard-coded with a specific value. It's fine to have that specific username in the researcher_reports.spec.js file, but not at this lower level.